packages/core/src/agent/MessageSender.ts

```diff
@@ -176,7 +176,7 @@ export class MessageSender {
     // If the other party shared a queue service endpoint in their did doc we queue the message
     if (queueService) {
```

Comment:
Ah yes, I remember this from https://github.com/hyperledger/aries-framework-javascript/issues/1199. I think it will be important to find a way to configure this. I'm trying to find the best place to update the code to support this, as I don't want to make `MessageSender` dependent on Mediator or Pickup modules API (we'll have some circular dependencies).